apps/src/componentLibrary/link/Link.tsx

```diff
@@ -5,27 +5,71 @@ import {ComponentSizeXSToL} from '@cdo/apps/componentLibrary/common/types';
 import moduleStyles from './link.module.scss';
 
 export interface LinkProps {
+  /** Link content */
+  children: React.ReactNode;
+  /** Link id */
+  id?: string;
+  /** Custom class name */
+  className?: string;
+  /** Does the link go to an external source? */
+  external?: boolean;
+  /** Should the link open in a new tab? */
+  openInNewTab?: boolean;
+  /** Link destination */
+  href: string;
+  /** Is the link disabled? */
+  disabled?: boolean;
+  /** Callback for click event */
+  onClick?: (event: React.MouseEvent<HTMLAnchorElement>) => void;
   /** Size of link */
   size?: ComponentSizeXSToL;
+  /** Color/type of link */
+  color?: 'primary' | 'secondary';
```

Comment:
`color` being either "primary" or "secondary" reads a little confusing to me. Would `type` make more sense?

Reply:
Makes sense, thanks for the catch!